charts/cinder-csi-plugin/values.yaml

```diff
@@ -31,7 +31,7 @@ csi:
   livenessprobe:
     image:
       repository: k8s.gcr.io/sig-storage/livenessprobe
-      tag: v2.1.0
+      tag: v2.2.0
```

Comment:
You might have known this already, either way — this change had great impact, in resolving this memory leak: https://github.com/kubernetes-csi/livenessprobe/pull/94

Reply:
thanks for the info, seems this is not memory leak but memory usage high 
but anyway, glad it fixed the problem and we adopted it